projects/transformers/trainer_mixins/lr_range_test.py

```diff
@@ -0,0 +1,97 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2021, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+from torch.optim.lr_scheduler import LambdaLR
+
+
+class LRRangeTestMixin:
```

Comment:
I read the code and the docstring, but I gotta say have no idea how is this mixin calculating a LRRange or how it is supposed to be used.

Reply:
I've updated the docstring. Please let me know what you think. It's much more explicit in how to use it.

Reply:
I am still confused on how to use it, maybe an example could help in a future PR. The example information might me more suited in an experiment file as well. We can talk about it later.